bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToZATo2L2B_ggfusion_b-associatedproduction/HToZATo2L2B_1000p00_200p00_0p50_bbH4F_TuneCP5_13TeV-amcatnloFXFX-pythia8/HToZATo2L2B_1000p00_200p00_0p50_bbH4F_TuneCP5_13TeV-amcatnloFXFX-pythia8_run_card.dat

```diff
@@ -0,0 +1,196 @@
+  1.0   = mur_over_ref ! ratio of current muR over reference muR
+  1.0   = muf_over_ref ! ratio of current muF over reference muF 
+#*********************************************************************** 
+# Reweight variables for scale dependence and PDF uncertainty          *
+#***********************************************************************
+  1.0, 2.0, 0.5	= rw_rscale ! muR factors to be included by reweighting
+  1.0, 2.0, 0.5	= rw_fscale ! muF factors to be included by reweighting
+  True	= reweight_scale ! Reweight to get scale variation using the 
+            ! rw_rscale and rw_fscale factors. Should be a list of 
+            ! booleans of equal length to dynamical_scale_choice to
+            ! specify for which choice to include scale dependence.
+  False	= reweight_pdf ! Reweight to get PDF uncertainty. Should be a
+            ! list booleans of equal length to LHAPDF ID to specify for
+            !  which PDF set to include the uncertainties.
+  -1.0	= rw_fscale_down  ! lower bound for fact scale variations
+  -1.0	= rw_rscale_down  ! upper bound for fact scale variations
+  -1.0	= rw_fscale_up   ! upper bound for fact scale variations
+  -1.0	= rw_rscale_up   ! upper bound for ren scale variations
+  325501   = PDF_set_min      ! First of the error PDF sets 
+  325600   = PDF_set_max      ! Last of the error PDF sets
+#************************************************************************
+## Renormalization and factorization scales (advanced and NLO options)  *
+##***********************************************************************
+  False	= fixed_qes_scale     ! if .true. use fixed Ellis-Sexton scale
+  91.118	= qes_ref_fixed   ! fixed Ellis-Sexton reference scale
+  1.0	= mur_over_ref ! ratio of current muR over reference muR
+  1.0	= qes_over_ref ! ratio of current QES over reference QES
+#***********************************************************************
+# Store reweight information in the LHE file for off-line model-       *
+# parameter reweighting at NLO+PS accuracy                             *
+#***********************************************************************
+  True	= store_rwgt_info ! Store info for reweighting in LHE file
+#***********************************************************************
+# ickkw parameter:                                                     *
+#   0: No merging                                                      *
+#   3: FxFx Merging - WARNING! Applies merging only at the hard-event  *
+#      level. After showering an MLM-type merging should be applied as *
+#      well. See http://amcatnlo.cern.ch/FxFx_merging.htm for details. *
+#   4: UNLOPS merging (with pythia8 only). No interface from within    *
+#      MG5_aMC available, but available in Pythia8.                    *
+#  -1: NNLL+NLO jet-veto computation. See arxiv:1412.8408 [hep-ph].    *
+#***********************************************************************
+  3	= ickkw 
```

Comment:
Do you have additional jets at NLO? Don't see that in the proc card.

Reply:
yes h3 decay to b b~ i add it in madspin_card for both NLO and LO